src/main/java/com/uber/profiling/Arguments.java

```diff
@@ -152,71 +156,114 @@ public void updateArguments(Map<String, List<String>> parsedArgs) {
             throw new RuntimeException("Sample interval too short, must be 0 (disable sampling) or at least " + Arguments.MIN_INTERVAL_MILLIS);
         }
 
-        tag = getArgumentSingleValue(parsedArgs, ARG_TAG);
-        logger.info("Got argument value for tag: " + tag);
+        argValue = getArgumentSingleValue(parsedArgs, ARG_TAG);
+        if (needToUpdateArg(argValue)) {
+            tag = argValue;
+            logger.info("Got argument value for tag: " + tag);
```

Comment:
debug or info?

Reply:
It is intended to be Info, want to be able to see the values in normal log